stellar-contract-env-common/src/env.rs

```diff
@@ -151,6 +154,29 @@ macro_rules! call_macro_with_all_host_functions {
                 {"$L", fn bigint_to_i64(x:Object) -> i64}
                 {"$M", fn bigint_from_i64(x:i64) -> Object}
             }
+
+            mod binary "x" {
+                {"$_", fn serialize_to_binary(x:Object) -> Object}
+                {"$0", fn deserialize_from_binary(x:Object) -> Object}
+                {"$1", fn binary_new() -> Object}
+                {"$2", fn binary_get(x:Object,i:RawVal) -> Object}
+                {"$3", fn binary_pop(x:Object) -> Object}
+                {"$4", fn binary_push(x:Object,v:RawVal) -> Object}
+                {"$5", fn binary_insert(x:Object,i:RawVal,v:RawVal) -> Object}
+                {"$6", fn binary_remove(x:Object,i:RawVal) -> Object}
+                {"$7", fn binary_len(x:Object) -> RawVal}
```

Comment:
❓ Could we add functions for binary to parallel vec? For example: slice, append, put, etc.

Reply:
Yes.

Reply:
This is updated. Now the binary functions mirror vec ones.